onnxruntime/core/framework/error_code.cc

```diff
@@ -15,7 +15,8 @@ struct OrtStatus {
 ORT_API(OrtStatus*, OrtCreateStatus, OrtErrorCode code, _In_ const char* msg) {
   assert(!(code == 0 && msg != nullptr));
   size_t clen = strlen(msg);
-  OrtStatus* p = reinterpret_cast<OrtStatus*>(new char[sizeof(OrtStatus) + clen]);
+  OrtStatus* p = reinterpret_cast<OrtStatus*>(::malloc(sizeof(OrtStatus) + clen));
+  if (p == nullptr) return nullptr;  // OOM
```

Comment:
if the purpose of this change is just to make new not throw then you can use nothrow too...